### PR TITLE
Added dockerfile for docker deployment with Nginx.

### PR DESCRIPTION
**Pull request by Bengrunt**
Added a simple `dockerfile` to build this game and get the generated files into an Nginx container that can serve them right away.

🌴

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+FROM node:alpine3.11
+
+COPY . /app
+WORKDIR /app
+
+RUN yarn && yarn build
+
+FROM nginx:stable-alpine
+LABEL author="Benjamin Guillon <bengrunt@gmail.com>"
```
Comment (adngdb) [inline Dockerfile:9]: That is some shameless name dropping! When this game is a hit, you'll be riding on its fame!

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+COPY --from=0 /app/public /usr/share/nginx/html
+
+# support running as arbitrary user which belongs to the root group
+RUN chmod g+rwx /var/cache/nginx /var/run /var/log/nginx
+
+# users are not allowed to listen on priviliged ports
```
Comment (adngdb) [inline Dockerfile:16]: s/priviliged/privileged

Comment (Bengrunt) [reply Dockerfile]: Fixed.

File: Dockerfile (hunk 1 above)
Comment (Bengrunt) [reply Dockerfile]: Fixed.

**Pull request merged**